__init__.py

```diff
@@ -11,7 +11,7 @@
 fallback_lang = "en_GB"
 def getLocale(lang: str):
     try:
-        with open(addon_dir / "locale" / f"{lang}.ftl") as f:
+        with open(addon_dir / "locale" / f"{lang}.ftl", encoding="utf-8") as f:
```

Comment:
@Luc-Mcgrady this was causing the same issue than for the other two lines. `UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 9115: ordinal not in range(128)`